Source/Core/EarthOrientationParameters.js

```diff
@@ -135,7 +135,7 @@ define([
      * Gets a promise that, when resolved, indicates that the EOP data has been loaded and is
      * ready to use.
      *
-     * @returns {Promise} The promise.
+     * @returns {Promise.<Object>} The promise.
```

Comment:
Currently, this promise resolves to no value, i.e. `undefined`.  We use this pattern in a few places to indicate that some process has finished, but there is no obvious result value of the process.

This particular class is marked "private", though, and is considered undocumented, so I don't think we need to worry about it for now.

In general, perhaps we should perhaps always try to return some value from all promises, even when not really necessary.  Perhaps as you suggest this loading promise could resolve to the EarthOrientationParameters object, even though the caller must have already had a reference to that object in order to access the promise in the first place.